.github/workflows/release.yml

```diff
@@ -0,0 +1,62 @@
+name: Add Releases
+
+on:
+  push:
+    # Pattern matched against refs/tags
+    tags:        
+      - '**' # Push events to every tag including hierarchical tags like v1.0/beta
+
+jobs:
+  draft:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Create Draft Release
+      uses: actions/create-release@v1
+      id: create_release
+      with:
+        draft: true
+        release_name: (Draft) Release ${{ github.event.release.tag_name }}
+        tag_name: ${{ github.ref }}
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
+
+    outputs:
+      release-upload-url: ${{ steps.create_release.outputs.upload_url }}
+
+  build:
+    runs-on: ubuntu-latest
+    needs: draft
+    strategy:
+      matrix:
+        lang: ['fr', 'es-la', 'pt-br', 'en']
```

Comment:
add lang `eu`

Reply:
added in https://github.com/ArdaxHz/mupl/pull/21/commits/b59a300decd3ba39f0b5b784f99cd2d0d0e789c4